packages/web/src/utils/index.js

```diff
@@ -169,3 +169,30 @@ export const getPopularSuggestionsComponent = (data = {}, props = {}) => {
 	}
 	return null;
 };
+
+// to check whether arrays are equal in values
+// irrespective of the order of entries
+export const areArraysEqual = (arr1, arr2) => {
```

Comment:
@mohdashraf010897 Remove this because you're not using it.

Reply:
👍